src/main/java/com/xpanxion/java/springboot/da1/demo/controller/student1/ManagerController1.java

```diff
@@ -0,0 +1,45 @@
+package com.xpanxion.java.springboot.da1.demo.controller.student1;
+
+import com.xpanxion.java.springboot.da1.demo.model.student1.Manager;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class ManagerController1 {
+
+    @Autowired
+    DataAccess dataAccess;
+
+    @GetMapping("student{id}/api/v1/managers")
+    public List<Manager> getManagers(@PathVariable("id") Integer id) {
+        var managersList = dataAccess.getManagers();
+        return managersList;
+    }
+
+    @Service
+    public static class DataAccess {
```

Comment:
Can you change the name of this to DataAccess1?  We are having name collisions with Spring Boot, anything with a Spring Boot annotation is will potentially collide with another student.

Reply:
Changing now